src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -25,15 +25,17 @@ the main body of the message as well as a quote.
 </docs>
 
 <template>
-	<div
+	<li
 		:id="`message_${id}`"
 		ref="message"
 		class="message"
 		:class="{'hover': showActions && !isSystemMessage, 'system' : isSystemMessage}"
 		@mouseover="showActions=true"
 		@mouseleave="showActions=false">
 		<div v-if="isFirstMessage && showAuthor" class="message__author">
-			<h6>{{ actorDisplayName }}</h6>
+			<h6 role="heading" aria-level="2">
```

Comment:
Btw I’d bet that we are skipping some heading levels here like h4 and h5, which is semantically incorrect. We should use the level which makes sense for the markup, and then style it to fit how we want it to look like.

Reply:
yeah, seems h6 was used in many places mostly for formatting purposes.

I could make it a h2 if it looks identical

Reply:
Should it be h2 though? Isn’t it:
1. h1 is the title "Nextcloud" or rather "Nextcloud Talk"
2. h2 would e.g. be the room name
3. h3 would be stuff like the message author

And then `.message__author h3` would need styling to make it look as before.

Reply:
@jancborchardt I've used one heading level for the day section as it's useful for grouping and navigation

Reply:
I've moved away from using "<h?>"  tags as I didn't want to fight against the styles trying to reproduce the existing style.

Instead, I used the alternative form of adding `role='heading'  aria-level='x'` to define the heading, which is valid and will be parsed as such by screen readers.

The hierarchy is now:
1. h1 app title
2. h2 room name
3. h3 date separator for message list grouped by date (useful for jumping to a different date)
4. h4 message author and message quote author

On the date item on the right I simply used a span as it's not really a header.